traffic_monitor/experimental/traffic_monitor/manager/datarequest.go

```diff
@@ -198,6 +204,16 @@ func NewCacheStatFilter(params url.Values, cacheTypes map[enum.CacheName]enum.Ca
 	}, nil
 }
 
+// This is the "spirit" of how TM1.0 works; hack to extract a path argument to filter data (/publish/SomeEndpoint/:argument).
+func getPathArgument(path string) string {
+	pathParts := strings.Split(path, "/")
+	if len(pathParts) == 4 {
```

Comment:
I don't have strong feelings on `>=` vs `==`, but I do have strong feelings that _if_ we're rejecting extra path params, we should definitely log and/or return an error explaining why the filter was rejected; ideally both.